tools/e2e/examples/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,195 @@
+    # However, `ports` settings are incompatible with `network_mode: service:...`
+    # and had to be moved here.
+    ports:
+    - "8000:8000"
+    networks:
+      kuma-example:
+        aliases:
+        - kuma-example-app
+    depends_on:
+    - kuma-control-plane
+    restart: on-failure
+
+  #
+  # Example application deployed into Kuma mesh.
+  #
+  # Notice that we're using Kong (API Gateway) as an example application
+  # since it's a perfect example of a (micro-)service that is both
+  # source and destination for network requests.
+  #
+  kuma-example-app:
+    image: kong:1.1.2-alpine
+    volumes:
+    - ./kong.yaml:/etc/kong/kong.yaml
+    environment:
+    - KONG_PROXY_LISTEN=127.0.0.1:8000
+    - KONG_ADMIN_LISTEN=off
+    - KONG_DATABASE=off
+    - KONG_DECLARATIVE_CONFIG=/etc/kong/kong.yaml
+    # the following setting instructs Docker Compose to add `kuma-example-app` container
+    # to the network namespace of `kuma-example-app-namespace` container
+    network_mode: service:kuma-example-app-namespace
+    depends_on:
+    - kuma-example-app-namespace
+    restart: on-failure
+
+  #
+  # Kuma sidecar for the example application.
+  #
+  kuma-example-app-sidecar:
+    # image name must be provided via environment variable KUMA_DP_DOCKER_IMAGE
+    image: ${KUMA_DP_DOCKER_IMAGE:-kuma/kuma-dp:latest}
+    command:
+    - run
+    - --log-level=info
+    environment:
+    - KUMA_CONTROL_PLANE_BOOTSTRAP_SERVER_URL=http://kuma-control-plane:5682
+    - KUMA_DATAPLANE_MESH=default
+    - KUMA_DATAPLANE_NAME=kuma-example-app
+    - KUMA_DATAPLANE_ADMIN_PORT=9901
+    # the following setting instructs Docker Compose to add `kuma-example-app-sidecar` container
+    # to the network namespace of `kuma-example-app-namespace` container
+    network_mode: service:kuma-example-app-namespace
+    depends_on:
+    - kuma-example-app-namespace
+    restart: on-failure
+
+  #
+  # Auxiliary service (container) for sharing Linux network namespace
+  # between Example Client and Kuma Sidecar.
+  #
+  kuma-example-client-namespace:
+    image: busybox:1.31.0
+    command: ["sleep", "infinity"]
+    # The following `expose` configuration was originally a part of `kuma-example-client` service.
+    # However, `expose` settings are incompatible with `network_mode: service:...`
+    # and had to be moved here.
+    expose:
+    - "3000"
+    # The following `ports` configuration was originally a part of `kuma-example-client` service.
+    # However, `ports` settings are incompatible with `network_mode: service:...`
+    # and had to be moved here.
+    ports:
+    - "3000:3000"
+    networks:
+      kuma-example:
+        aliases:
+        - kuma-example-client
+    depends_on:
+    - kuma-control-plane
+    restart: on-failure
+
+  #
+  # Example client deployed into Kuma mesh.
+  #
+  kuma-example-client:
+    image: kong-docker-kuma-ci-docker.bintray.io/curl
```

Comment:
future us should move these off of bintray

Reply:
@hutchic Sorry, I don't get it. Why ?

Reply:
cost mostly but also reliability